src/UIKit/UIGestureRecognizer.cs

```diff
@@ -319,4 +319,27 @@ public void Activated (UIScreenEdgePanGestureRecognizer sender)
 #endif
 }
 
+// start code for UIHoverGestureRecognizer Constructor
+	public partial class UIHoverGestureRecognizer : UIGestureRecognizer {
+		public UIHoverGestureRecognizer (Action action) : base (action) {}
```

Comment:
I believe this is not needed since we already expose a constructor in the generated code:
```csharp
	[NoWatch, NoTV, iOS (13,0)]
	[BaseType (typeof (UIGestureRecognizer))]
	[DisableDefaultCtor]
	interface UIHoverGestureRecognizer {

		[DesignatedInitializer]
		[Export ("initWithTarget:action:")]
		NativeHandle Constructor (NSObject target, Selector action);

		[DesignatedInitializer]
		[Wrap ("base (action)")]
		NativeHandle Constructor (Action action);
	}
```
Have your tried removing it? Is it giving you an issue?